### PR TITLE
V6.0.0 rc

### PR DESCRIPTION
**Pull request by doug-martin**
* Updated go support to `1.10`, `1.11` and `1.12`
* Change testify dependency from c2fo/testify back to stretchr/testify.
* Add support for "FOR UPDATE" and "SKIP LOCKED" [#62](https://github.com/doug-martin/goqu/pull/62) - [@btubbs](https://github.com/btubbs)
* Changed to use go modules

### CODE REVIEW

File: example_test.go [hunk 1]
```diff
@@ -614,8 +614,8 @@ func ExampleDataset_WithCTERecursive() {
 	sql, _, _ := db.From("nums").
 		WithRecursive("nums(x)",
 			db.From().Select(goqu.L("1")).
-			UnionAll(db.From("nums").
-				Select(goqu.L("x+1")).Where(goqu.I("x").Lt(5)))).
+				UnionAll(db.From("nums").
+					Select(goqu.L("x+1")).Where(goqu.I("x").Lt(5)))).
```
Comment (funkyshu) [inline example_test.go]: i think maybe go fmt would make these line not staggered.  I might be wrong but might wanna make sure you run go fmt against the project.

File: adapters/mysql/mysql_test.go [hunk 2]
```diff
@@ -3,14 +3,14 @@ package mysql
 import (
 	"database/sql"
 	"fmt"
+	"github.com/doug-martin/goqu"
```
Comment (funkyshu) [inline adapters/mysql/mysql_test.go]: Seeing this all over.  Standard library packages should be in their own group.  See run goimports
see https://github.com/golang/go/wiki/CodeReviewComments#imports

Might wanna runn goimports

File: example_test.go (hunk 1 above)
Comment (doug-martin) [reply example_test.go]: Yeah it didnt reformat that section

File: adapters/mysql/mysql_test.go (hunk 2 above)
Comment (doug-martin) [reply adapters/mysql/mysql_test.go]: done

**Pull request merged**